test/test_prototype_transforms_functional.py

```diff
@@ -421,6 +421,14 @@ def center_crop_bounding_box():
         )
 
 
+def center_crop_segmentation_mask():
+    for mask, output_size in itertools.product(
+        make_segmentation_masks(),
+        [[4, 3], [42, 70], [4]],  # crop sizes < image sizes, crop_sizes > image sizes, single crop size
```

Comment:
Should we maybe not rely on the default values here to make sure we actually get the cases detailed in the comment?

Reply:
I still use the default values, but I have made them explicit.